RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx

```diff
@@ -1060,7 +1060,7 @@ console.log(createdBoard);
           <SelectorCombo<WebApiTeam>
             className="team-selector"
             currentValue={this.state.currentTeam}
-            iconName="People"
+            iconName="users"
```

Comment:
Is this a change in the icons available from the updated libraries?

Reply:
yes, existing codebase heavily uses fluent ui icons (which adds almost a mb to the payload), and fontawesome icons (which adds around 500kb to the payload)

I'd like to get rid of both of the icon packs and use only svg icons.

to be more future compatible and ready to remove fluent icons, I modified some of the ui elements. `SelectorCombo` component is one of them.

it was using `Icon` component from fluent icon package, now it has regular dom element to show icons, that's why I need to change `iconName` properties

with these changes, we have less dom element, more ready to completely remove the fluent ui package, etc.